test/Makefile

```diff
@@ -1,24 +1,29 @@
-REPO := $(shell basename $(shell git remote get-url origin) .git)
-CONTAINER := $(REPO)_test
+ACCOUNT := 224588347132
+CONTAINER := terraform_modules_test
```

Comment:
Let's refactor this to match what is done in the lb module. See my notes there.

Reply:
Check.